packages/suite/src/views/wallet/coinmarket/transactions/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { withSelectedAccountLoaded } from 'src/components/wallet';
+import { CoinmarketAccountTransactions } from '../common/CoinmarketLayout/CoinmarketAccountTransactions/CoinmarketAccountTransactions';
+import { useLayout } from 'src/hooks/suite';
+import { PageHeader } from 'src/components/suite/layouts/SuiteLayout';
+
+const CoinmarketTransactions = () => {
+    useLayout('Trezor Suite | Trade', () => <PageHeader backRoute="wallet-coinmarket-buy" />);
+
+    return <CoinmarketAccountTransactions />;
+};
+
+export default withSelectedAccountLoaded(CoinmarketTransactions, {
```

Comment:
We no longer do index files and default exports. 

I assume filename `CoinmarketTransactions.tsx` with:

```
export const CoinmarketTransactions = withSelectedAccountLoaded(_CoinmarketTransactions, {
```